Nextjs

Mazayw opened the pull request.

NataPas2021 commented on README.md:
12 строка - там, мабуть, має бути npm dev чи нє?

Mazayw replied:
Так, дякую)

Mazayw committed:
Add project (#116)

* fix: translation

* fix: language selector appearence on mobile

* feat: add hyst

Neschadin committed:
fix: refactor ProjectForm; add defaultValues;

Fstv-Electro committed:
feat: change useHeroSliderSWR (add mutate)

CHANGE BREAK: need fix file upload onSubmit

Neschadin committed:
fix: start developing ProjectTeamList;

Neschadin committed:
fix: continued developing ProjectMemberList;

Fstv-Electro committed:
fix: refactoring form code and add DefaultValues from enother branch

CHANGE BREAK: need add validation to inputs, fix text block in preview

Fstv-Electro committed:
fix: post 'CreatNewSlider'

CHANGE BREAK: need fix double http post

SlipuyKamiN committed:
Merge branch 'nextjs' into add-partners-on-adminpage

Fstv-Electro committed:
fix: fix 'add new slider', change validation rules

CHANGE BHREAK: need check double post requests

Fstv-Electro committed:
fix: url in editSlider

Neschadin committed:
fix: refactor Forms;

SlipuyKamiN committed:
fix: routing logic, global refactoring

Fstv-Electro committed:
fix: useHeroSliderSWR > updateSlider, onSubmit

CHANGE BREAK: need fix preview text container

Fstv-Electro committed:
fix: preview text container

Fstv-Electro committed:
fix: add controller to inputs, change SliderDeleteButton, change SliderPreview logic out render

Fstv-Electro committed:
fix: connect to main page slides from backend

Neschadin committed:
fix: fix;

Neschadin committed:
feat: upd;

Neschadin committed:
Merge branch 'nextjs' into nesh_admin_projects_page

Neschadin committed:
Merge branch 'nesh_admin_projects_page' of https://github.com/baza-trainee/baza-trainee-landing into nesh_admin_projects_page

Mazayw committed:
Merge pull request #101 from baza-trainee/nesh_admin_projects_page

Nesh admin projects page

Fstv-Electro committed:
Merge branch 'nextjs' into admin/slider

Fstv-Electro committed:
fix: fix;

Fstv-Electro committed:
fix: fix file input placeholder value

Neschadin committed:
fix: fix ProjectCard image;

Fstv-Electro committed:
Merge pull request #119 from baza-trainee/nesh_admin_projects_page

fix: fix ProjectCard image;

Fstv-Electro committed:
fix: fix types, Image, api url, background languageSelector, double export, handleError SWR, shadow

Fstv-Electro committed:
fix: fix validateOption for slider photo

Fstv-Electro committed:
Admin/slider (#118)

* feat: add AdminPage heroSlider

* feat: add styles to admin page heroSlider

CHANGE BREAKER: find solution for height tailwind

* fix: change height and width inline to TW and add DeleteIcon in Slider index.tsx

* feat: add AdminSingleSlide component

CHANGE BREAKER: need add buttons and logic for icons

* feat: add links, delete slide component, add 'add-slider', 'edit-slider' pages

CHANGE BREAKER: add formSlider component, preview image and check axios

* feat: add FomrBtns from: Neschadin's branch

* fix: useHeroSliderSWR

CHANGE BREAKER: need connect useHeroSliderSWR and test

* feat: add locales, dictionalries, errorHandler from enother branch

fix: GlobalContext, useHeroSlidersSWR

* feat: init SliderForm and hook-form npm

BREAK CHANGE: need fix props and types in form

* fix: changed from another branch FileInput and TextInputField

CHANGE BREAK: need add slideReview

* feat: add preview img

CHANGE BREAK: need add text to preview

* fix: add preview backdrop, text. Change img to Image

CHANGE BREAK: need add code for Edit Form and change route for current language

* fix: useGlobalContext in files

* feat: change useHeroSliderSWR (add mutate)

CHANGE BREAK: need fix file upload onSubmit

* fix: refactoring form code and add DefaultValues from enother branch

CHANGE BREAK: need add validation to inputs, fix text block in preview

* fix: post 'CreatNewSlider'

CHANGE BREAK: need fix double http post

* fix: fix 'add new slider', change validation rules

CHANGE BHREAK: need check double post requests

* fix: url in editSlider

* fix: useHeroSliderSWR > updateSlider, onSubmit

CHANGE BREAK: need fix preview text container

* fix: preview text container

* fix: add controller to inputs, change SliderDeleteButton, change SliderPreview logic out render

* fix: connect to main page slides from backend

* fix: fix;

* fix: fix file input placeholder value

* fix: fix types, Image, api url, background languageSelector, double export, handleError SWR, shadow

* fix: fix validateOption for slider photo

Nataly-Moskalenko committed:
feat: adds styles to testimonials admin

Nataly-Moskalenko committed:
feat: deletes empty folder

Nataly-Moskalenko committed:
feat: fixes property: map in testimonials page

SlipuyKamiN committed:
build: v1.0, add functionality, no styles fixed

SlipuyKamiN committed:
fix: warnings

SlipuyKamiN committed:
feat: upd from "nextjs"

SlipuyKamiN committed:
fix: btn

SlipuyKamiN committed:
fix: fit to global changes

SlipuyKamiN committed:
fix: rerender, import bugs

SlipuyKamiN committed:
fix: validation color

SlipuyKamiN committed:
fix: remove unused keys

Fstv-Electro committed:
feat: add disabled when max slides

Fstv-Electro committed:
Merge branch 'nextjs' into admin/slider

Fstv-Electro committed:
fix: projects title, some locales, change sliders data from local

Nataly-Moskalenko committed:
feat: fixes styles, adds edit link to testimonials admin

Mazayw committed:
Merge pull request #121 from baza-trainee/admin/slider

Admin/slider

SlipuyKamiN committed:
fix: describe types, refactoring, fix pagination

Nataly-Moskalenko committed:
feat: fixes testimonials swr

Nataly-Moskalenko committed:
Merge branch 'nextjs' of https://github.com/baza-trainee/baza-trainee-landing into feature/admin-testimonials

Fstv-Electro committed:
feat: add modal delete for slides

Nataly-Moskalenko committed:
feat: creates main-testimonials component

Fstv-Electro committed:
fix: fix SWR after update backend

Fstv-Electro committed:
fix: fix url Image in admin signle slide

Nataly-Moskalenko committed:
feat: adds formatted date to single-slide-testimonial

Fstv-Electro committed:
fix: change FormBtns to reusable component

SlipuyKamiN committed:
fix: little bugs, refactoring

SlipuyKamiN committed:
fix: add dependencies

SlipuyKamiN committed:
fix: warnings

SlipuyKamiN committed:
fix: add trim to inputs

SlipuyKamiN committed:
fix: pagination

Fstv-Electro committed:
fix: change FormBtns like 'dumb' button, change project, member and role form passing func prop

Neschadin committed:
Merge pull request #125 from baza-trainee/admin/slider

Admin/slider

nadiiahonchar1 committed:
feat: add settings

Fstv-Electro committed:
fix: change SWR response and type

SlipuyKamiN committed:
Merge branch 'nextjs' into add-partners-on-adminpage

nadiiahonchar1 committed:
feat: add change

nadiiahonchar1 committed:
fix: del inputs login password & add change password

Neschadin committed:
fix: fix;

Neschadin committed:
Merge branch 'nextjs' into nesh_admin_projects_page

SlipuyKamiN committed:
build: add image dimensions validation

SlipuyKamiN committed:
fix: add svg to accept image formats

Nataly-Moskalenko committed:
refactor: tslide-review and single-slide in reviews

SlipuyKamiN committed:
fix: change image dimension requirments

Nataly-Moskalenko committed:
Merge branch 'nextjs' of https://github.com/baza-trainee/baza-trainee-landing into feature/admin-testimonials

Nataly-Moskalenko committed:
fix: use-testimonials-swr

Neschadin committed:
fix: continued developing;

Neschadin committed:
fix: fix ButtonOverlay; deploy link; n some fixs;

nadiiahonchar1 committed:
fix: add status button

nadiiahonchar1 committed:
fix: add button type to InputField

Neschadin committed:
Update useHeroSlidersSWR.ts

Neschadin committed:
Merge pull request #126 from baza-trainee/admin/slider

fix: change SWR response and type

Neschadin committed:
feat: implemented role sorting logic;

nadiiahonchar1 committed:
fix: add value

nadiiahonchar1 committed:
Merge branch 'nextjs' of https://github.com/baza-trainee/baza-trainee-landing into nextjs-settings

Neschadin committed:
fix: action FormBtns;

Neschadin committed:
Merge branch 'nextjs' into nesh_admin_projects_page

Neschadin committed:
fix: fix;

nadiiahonchar1 committed:
fix: add interface IChangePasswordRequest

nadiiahonchar1 committed:
fix: add request

Fstv-Electro committed:
fix: remove backdrop from AdminSingleSlide and PreviewSlide

Mazayw committed:
fix: members links

Neschadin committed:
Merge pull request #129 from baza-trainee/fixes_ihor

fix: members links

nadiiahonchar1 committed:
fix: add validation

nadiiahonchar1 committed:
fix: del status 404

nadiiahonchar1 committed:
fix: removed redundant validation

kholodihor committed:
feat: add create and edit pages

Mazayw committed:
Merge pull request #127 from baza-trainee/nextjs-settings

Nextjs settings

kholodihor committed:
fix: turn main page testimonials slider to server data

Neschadin committed:
feat: implement FormProvider;

SlipuyKamiN committed:
Merge branch 'nextjs' into add-partners-on-adminpage

SlipuyKamiN committed:
fix: input file change bug

Neschadin committed:
feat: finish;

Neschadin committed:
Merge branch 'nextjs' into nesh_admin_projects_page

Neschadin committed:
fix: fix btns prop in Settings;

Neschadin committed:
fix: fix method name useProjectSWR;

Fstv-Electro committed:
fix: text center in slides

nadiiahonchar1 committed:
feat: add ReguestReset

nadiiahonchar1 committed:
fix: added link to Autorization

Mazayw committed:
Merge pull request #128 from baza-trainee/nesh_admin_projects_page

Nesh admin projects page

Neschadin committed:
fix: fix members link to profileUrl;

Neschadin committed:
feat: transfer to new ProjectCard;

Neschadin committed:
fix: projects title;

Neschadin committed:
fix: high project title;

Neschadin committed:
fix: title ProjectCard;

Mazayw committed:
Merge pull request #132 from baza-trainee/nesh_admin_projects_page

Nesh admin projects page

kholodihor committed:
feat: add swr to testimonials

Neschadin committed:
fix: edit project mode;

Neschadin committed:
fix: fix;

Mazayw committed:
fix: typo

Mazayw committed:
Merge pull request #134 from baza-trainee/fixes_ihor

fix: typo

SlipuyKamiN committed:
fix: file validation error message

SlipuyKamiN committed:
fix: preview images size

SlipuyKamiN committed:
fix: search handling, preview imgs width

SlipuyKamiN committed:
Merge branch 'nextjs' into add-partners-on-adminpage

Neschadin committed:
fix: fix FileInput, ProjectCard;

Neschadin committed:
fix: fix SliderForm;

Neschadin committed:
fix: del register;

Neschadin committed:
fix: project members after mutate;

Neschadin committed:
fix: projectPreview;

Neschadin committed:
fix: reset pass form & btns;

nadiiahonchar1 committed:
feat: add passwordReset

kholodihor committed:
refactor: remove unnecessary code

nadiiahonchar1 committed:
fix: resolved types conflict

Mazayw committed:
Merge pull request #89 from baza-trainee/add-partners-on-adminpage

Add partners on adminpage

Mazayw committed:
Merge pull request #131 from baza-trainee/nextjs-password-reset

Nextjs password reset

kholodihor committed:
feat: merged changes from remote HEAD

Mazayw committed:
Merge pull request #130 from baza-trainee/feature/admin-testimonials

Feature/admin testimonials

Neschadin committed:
fix: to check deploy;

Neschadin committed:
Merge branch 'nextjs' into new_nesh_projects

Neschadin committed:
fix: fix;

Neschadin committed:
fix: prev state;

Neschadin committed:
feat: update packages;

Mazayw committed:
Merge pull request #135 from baza-trainee/new_nesh_projects

New branch nesh projects

Neschadin committed:
fix: fix SWR hooks;

Mazayw committed:
Merge pull request #136 from baza-trainee/new_nesh_projects

New nesh projects

Fstv-Electro committed:
Merge branch 'nextjs' into admin/slider

Mazayw committed:
Merge pull request #137 from baza-trainee/admin/slider

Admin/slider

kholodihor committed:
perf: updated translate logic in testimonials and added imports sorting

Mazayw committed:
Merge pull request #138 from baza-trainee/feature/admin-testimonials

perf: updated translate logic in testimonials and added imports sorting

Neschadin committed:
fix: some fixs; move some libs to devDeps;

Neschadin committed:
fix: move some libs to devDeps;

Neschadin committed:
feat: finish search member in ProjectEditMode;

Neschadin committed:
fix: fix;

Mazayw committed:
Merge pull request #139 from baza-trainee/new_nesh_projects

search member in ProjectEditMode;

Neschadin committed:
fix: fix actionBtns;

Neschadin committed:
fix: del logs;

Neschadin committed:
Merge pull request #140 from baza-trainee/new_nesh_projects

fix actionBtns;

Neschadin committed:
feat: full refactor Partner section;

Neschadin committed:
feat: finish refactor Partner section;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: del usePartnerForm hook:  clear types;

Neschadin committed:
fix: pagination bar;

Mazayw committed:
Merge pull request #141 from baza-trainee/nesh_partners

full reafactor partner section;

Neschadin committed:
fix: fix FormBtns;

Neschadin committed:
feat: switched partners to fetching data from DB;

Neschadin committed:
fix: fix bugs 91, 89, 92, 96, 97, 95;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: upd submit methods in Forms;

kholodihor committed:
fix: fixed testimonials validation

Neschadin committed:
feat: implemented img dimensions checking;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: validate project field name & partner logo;

kholodihor committed:
fix: added validation by image size and date validation

Fstv-Electro committed:
fix: connect input to translator

BREAKING CHANGE: need fix routes on changing language option

Fstv-Electro committed:
feat: commit for PR

Neschadin committed:
fix: refactor couple func;

Neschadin committed:
fix: fix;

Neschadin committed:
feat: implemented switch component lang;

Mazayw committed:
Merge pull request #144 from baza-trainee/admin/slider

Admin/slider

Commit:
fix: switched partners to mockData;

Commit:
Merge branch 'nextjs' into nesh_switcher_translator

Commit:
fix: fix;

Mazayw committed:
Merge pull request #145 from baza-trainee/nesh_switcher_translator

implemented switch component lang;

kholodihor committed:
fix: fix validations and translate

Commit:
fix: searching bug;

Commit:
fix: strange behavior of the language selector;

Neschadin committed:
fix: refactored some comps;

kholodihor committed:
Merge pull request #146 from baza-trainee/nesh_switcher_translator

fix strange behavior of the language selector;

kholodihor committed:
Merge remote-tracking branch 'origin/HEAD' into feature/admin-testimonials

kholodihor committed:
fix: fixed image validation when update

Fstv-Electro committed:
fix: add validation img dimensions, fix LanguageSelector props, add dimensions in SETTINGS

BREAKING CHANGE: need fix rerender after add/update/delete slide

Neschadin committed:
Merge pull request #147 from baza-trainee/admin/slider

fix: add validation img dimensions, fix LanguageSelector props, add dimensions in SETTINGS

Neschadin committed:
fix: optimize translate handler;

Neschadin committed:
Merge branch 'nextjs' into feature/admin-testimonials

Neschadin committed:
fix: some fix;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: validate value check condition;

Neschadin committed:
fix: eslintrc.json;

Neschadin committed:
Merge pull request #143 from baza-trainee/feature/admin-testimonials

fix: fixed testimonials validation

Neschadin committed:
feat: refactor admin slider;

Neschadin committed:
feat: finish refactor admin slider section;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: remove logs;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: fix;

Mazayw committed:
Merge pull request #148 from baza-trainee/nesh_some_fixs

refactor slider section;

Fstv-Electro committed:
feat: connect sliders to mainPage

kholodihor committed:
feat: switch MainPage Reviews to database

Neschadin committed:
Merge pull request #149 from baza-trainee/admin-to-main-sliders

feat: connect sliders to mainPage

Neschadin committed:
Update index.tsx

Neschadin committed:
Merge pull request #150 from baza-trainee/feature/admin-testimonials

testimonials switched to database data

Neschadin committed:
fix: fix project del func;

kholodihor committed:
fix: fixed member name validation

Neschadin committed:
Merge pull request #151 from baza-trainee/feature/admin-testimonials

fix: fixed member name validation

Neschadin committed:
fix: fix;

kholodihor committed:
Merge pull request #152 from baza-trainee/nesh_fix_projetcs

Nesh fix projetcs

Neschadin committed:
fix: swr keys;

kholodihor committed:
Merge pull request #153 from baza-trainee/nesh_fix_projetcs

fix: swr keys;

Fstv-Electro committed:
fix: fix src link for sliders

kholodihor committed:
Merge pull request #154 from baza-trainee/fix-sliders-src

fix: fix src link for sliders

Fstv-Electro committed:
fix: props data from mainPage

kholodihor committed:
Merge pull request #155 from baza-trainee/fix-sliders-src

fix: props data from mainPage

Neschadin committed:
fix: member validate; pagination logic; role eng;

Neschadin committed:
fix: lang of MembersAndRoles;

Neschadin committed:
fix:  throw Promise.reject()  in all SWR hooks;

Neschadin committed:
fix: fix;

Fstv-Electro committed:
Merge pull request #156 from baza-trainee/nesh_member_fix

fix: member validate; pagination logic; role eng;

Neschadin committed:
fix: fix;

Mazayw committed:
Merge pull request #157 from baza-trainee/nesh_member_fix

fix: fix;

Neschadin committed:
feat: implementation "more project" btn logic;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: fix;

Neschadin committed:
fix: fix;

Fstv-Electro committed:
Merge pull request #158 from baza-trainee/nesh_more_projects_btn

Nesh more projects btn

nadiiahonchar1 committed:
feat: added usePasswordManipulation

Mazayw committed:
Merge pull request #159 from baza-trainee/nextjs-password-hook

feat: added usePasswordManipulation

Fstv-Electro committed:
feat: test to connect

Fstv-Electro committed:
fix: delete animation, connect achievments to main

Fstv-Electro committed:
Merge pull request #160 from baza-trainee/achivments-connect-to-maon

Achivments connect to maon

Fstv-Electro committed:
fix: turned off autocomplete in all forms

Mazayw committed:
Merge pull request #161 from baza-trainee/project-members-forms-fix-autocomplete

fix: turned off autocomplete in all forms

Neschadin committed:
fix: projects pagination;

nadiiahonchar1 committed:
Merge pull request #162 from baza-trainee/nesh_proj_fix

fix: projects pagination;

Neschadin committed:
fix: submit;

Neschadin committed:
Merge branch 'nextjs' into nesh_proj_fix

Neschadin committed:
Merge pull request #163 from baza-trainee/nesh_proj_fix

fix submit;

Mazayw committed:
fix: translation

Neschadin committed:
Merge pull request #164 from baza-trainee/translation-fixes

fix: translation

Mazayw committed:
fix: transaltion
